packages/fluentui/react-northstar/src/components/Button/Button.tsx

```diff
@@ -254,6 +263,22 @@ const Button = compose<'button', ButtonProps, ButtonStylesProps, {}, {}>(
     className: buttonClassName,
     displayName: 'Button',
 
+    slots: {
+      content: ButtonContent,
+      icon: Box,
+      loader: Loader,
+    },
+
+    slotProps: {
+      content: props => ({
+        size: props.size,
+        as: 'span',
```

Comment:
This is weird as this should be defined in `ButtonContent`

Reply:
Agree, I will move it there...